cloudbuild.yaml

```
@@ -22,7 +22,7 @@ steps:
       path: /root
 
   - id: TESTING
-    name: 'gcr.io/cloud-builders/mvn'
+    name: 'openjdk:11.0.6-slim'
```

Wrong spot for that one. Should be
```suggestion
    name: 'maven:3.6-jdk-11'
```